scipy/special/_add_newdocs.py

```diff
@@ -13044,8 +13044,38 @@ def add_newdoc(name, doc):
     Notes
     -----
 
+    The log function used in the computation is the natural log.
```

Comment:
No blank line between the section's title and the text
```suggestion
    -----
    The log function used in the computation is the natural log.
```